surveys/views.py

```diff
@@ -28,7 +28,7 @@ class AgencyCreate(CreateView):
     form_class = survey_forms.AgencyCreateForm
     model = pldp_models.Agency
     template_name = "agency_create.html"
-    success_url = '/'
+    success_url = 'agencies-list'
```

Comment:
Does this need to be wrapped in a `reverse_lazy` call? I got redirected to `agencies/create/agencies-list`, which returns 404.

Reply:
Ee sorry I should have tested that! You're right.